test/features/lb/public_lb.feature

```diff
@@ -4,11 +4,12 @@ Feature: Test different tfvars configurations for the lb module with a public ti
     Background: Initialize testing for lb module with a public tier.
         
         Given terraform module 'lb'
-            | key   | value                     |
-            #-------|---------------------------|
-            | name  | "test-lb"                 |
-            | vpc   | "techservicesastest2-vpc" |
-            | tier  | "public"                  |
+            | key             | value                     |
+            #-----------------|---------------------------|
+            | name            | "test-lb"                 |
+            | vpc             | "techservicesastest2-vpc" |
+            | tier            | "public"                  |
+            | ip_address_type | "ipv4"                    |
```

Comment:
Please drop the "ipv4" here and elsewhere since it is now the default.

Reply:
I've dropped these two lines.